packages/jest-changed-files/src/git.js

```diff
@@ -13,44 +13,65 @@ import type {Options, SCMAdapter} from 'types/ChangedFiles';
 import path from 'path';
 import childProcess from 'child_process';
 
+const findChangedFilesUsingCommand = async (
+  args: Array<string>,
+  cwd: Path,
+): Promise<Array<Path>> => {
+  return new Promise((resolve, reject) => {
+    const child = childProcess.spawn('git', args, {cwd});
+    let stdout = '';
+    let stderr = '';
+    child.stdout.on('data', data => (stdout += data));
+    child.stderr.on('data', data => (stderr += data));
+    child.on('error', e => reject(e));
+    child.on('close', code => {
```

Comment:
wow, we really should use `execa` or something...

Reply:
Is this a refactor that you want me to include in my bug fix branch, or is it okay to just move the existing code about unchanged for now, and and backlog the `execa` change for later?

(`git grep` shows it up as a dependency of `os-locale` and `lerna` in `yarn.lock`, but isn't used directly by jest anywhere)

Reply:
Sticking it in the backlog is no problem at all, and will keep this PR focused. So just keep the `child_process` stuff for now 🙂